source/data-modeling/documents.txt

```diff
@@ -86,10 +86,11 @@ The document appears in MongoDB as follows:
 Additional Information
 ----------------------
 
+To learn more about the field types you can use in {+odm+} models, see
```

Comment:
i: [need 'that'](https://www.mongodb.com/docs/meta/style-guide/writing/use-that-which-correctly/)

```suggestion
To learn more about the field types that you can use in {+odm+} models, see
```